runelite-client/src/main/java/net/runelite/client/plugins/wintertodt/WintertodtPlugin.java

```diff
@@ -328,22 +327,55 @@ else if (messageNode.getValue().startsWith("The brazier has gone out."))
 		{
 			boolean shouldNotify = false;
 
-			switch (config.notifyCondition())
+			switch (interruptType)
 			{
-				case ONLY_WHEN_INTERRUPTED:
-					if (wasInterrupted)
+				case COLD:
+					switch (config.notifyCold())
 					{
-						shouldNotify = true;
+						case OFF:
```

Comment:
This all simplifies to:
```
WintertodtNotifyDamage notify = config.notifyCold();
shouldNotify = notify == ALWAYS || (notify == INTERRUPT && wasInterrupted)
```

And can maybe be combined with the snowfall/brazier stuff below too.

Reply:
I (and others in #dev) couldn't think of a good way to combine the 3 cases, but I made this change and otherwise if this looks good to you I'll squash these commits